projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -859,6 +907,102 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
+    importPath: SPARTACUS_CORE,
+    comment: `'${USER_NORMALIZER}' was removed. For replacement please use '${USER_ACCOUNT_NORMALIZER}' from '${SPARTACUS_USER_ACCOUNT}' and '${USER_PROFILE_NORMALIZER}' from '${SPARTACUS_USER_PROFILE}'.`,
+  },
+  // projects/core/src/user/connectors/user/user.adapter.ts
+  {
+    node: USER_ADAPTER,
+    importPath: SPARTACUS_CORE,
+    comment: `'${USER_ADAPTER}' was removed, please use '${USER_ACCOUNT_ADAPTER}' from '${SPARTACUS_USER_ACCOUNT_CORE}' and '${USER_PROFILE_ADAPTER}' from '${SPARTACUS_USER_PROFILE_CORE}'. Also there was method name change, for more details please look into 4.0 migration documentation.`,
+  },
+  // projects/core/src/user/connectors/user/user.connector.ts
+  {
+    node: USER_CONNECTOR,
+    importPath: SPARTACUS_CORE,
+    comment: `'${USER_CONNECTOR}' was removed, please use '${USER_ACCOUNT_CONNECTOR}' from '${SPARTACUS_USER_ACCOUNT_CORE}' and '${USER_PROFILE_CONNECTOR}' from '${SPARTACUS_USER_PROFILE_CORE}'. Also there was slighly change in method logic, for more details please look into 4.0 migration documentation.`,
+  },
+  // projects/core/src/user/facade/user.service.ts
+  {
+    node: USER_SERVICE,
+    importPath: SPARTACUS_CORE,
+    comment: `Many methods from '${USER_SERVICE}' were removed, for more details please look into 4.0 migration documentation.`,
+  },
+  // projects/core/src/model/misc.model.ts
+  {
+    node: USER_SIGN_UP_INTERFACE,
+    importPath: SPARTACUS_CORE,
+    comment: `'${USER_SIGN_UP_INTERFACE}' was removed, for replacement please use '${USER_SIGN_UP_INTERFACE}' from '${SPARTACUS_USER_PROFILE}'.`,
+  },
+  // projects/core/src/user/store/actions/index.ts
+  {
+    node: USER_ACTIONS,
+    importPath: SPARTACUS_CORE,
+    comment: `Following actions '${FORGOT_PASSWORD_EMAIL_ACTION}', '${RESET_PASSWORD_ACTION}', '${EMAIL_ACTIONS}', '${UPDATE_PASSWORD_ACTION}', '${USER_DETAILS_ACTION}' were removed. Logic was moved to '${SPARTACUS_USER}'.`,
+  },
+  // projects/core/src/user/store/user-state.ts
+  {
+    node: USER_DETAILS_STATE_INTERFACE,
+    importPath: SPARTACUS_CORE,
+    comment: `'${USER_DETAILS_STATE_INTERFACE}' was removed.`,
+  },
+  // projects/core/src/user/store/user-state.ts
+  {
+    node: USER_STATE_INTERFACE,
+    importPath: SPARTACUS_CORE,
+    comment: `Properties 'account' 'titles', and 'resetPassword' were removed from '${USER_DETAILS_STATE_INTERFACE}' interface.`,
+  },
+  // projects/core/src/user/user-transitional-tokens.ts
+  {
+    node: USER_PROFILE_FACADE_TRANSITIONAL_TOKEN,
+    importPath: SPARTACUS_CORE,
+    comment: `Abstract methods 'get', 'update', 'close' were removed from '${USER_PROFILE_FACADE_TRANSITIONAL_TOKEN}'.`,
+  },
+  // projects/core/src/user/user-transitional-tokens.ts
+  {
+    node: USER_REGISTER_FACADE_TRANSITIONAL_TOKEN,
+    importPath: SPARTACUS_CORE,
+    comment: `Abstract methods 'register', 'registerGuest', 'getTitles' were removed from '${USER_REGISTER_FACADE_TRANSITIONAL_TOKEN}'.`,
```

Comment:
I think it needs to be adjusted that the whole class got removed

Reply:
True, changed the comment.